bin/MadGraph5_aMCatNLO/cards/production/13TeV/HZaTo2l2g/HZaTo2l2g_run_card.dat

```diff
@@ -0,0 +1,270 @@
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************
+# Running parameters
+#*******************
+#
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .true.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  1000 = nevents ! Number of unweighted events requested
+      0       = iseed   ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+        1     = lpp1    ! beam 1 type
+        1     = lpp2    ! beam 2 type
+     6500     = ebeam1  ! beam 1 total energy in GeV
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set
+ 260000 = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
Why not use NNPDF31?

Reply:
This is because we want to produce signal samples consistent with re-reco. We will use 260000 for 2016 and 306000 for 2017-2018 for Legecy sample. Of course, we might also need UL sample in the future. In this case, we will use NNPDF31 for all three years.

Reply:
Are you also aware that transition to Ultra Legacy would be mandatory soon?

+ 260000 does not make sense to me as this is NLO PDF for LO hard process
(of course one might raise that NNPDF31NNLO is also mismatching but this at least is verified to be OK with CP5 tunes).

Reply:
Yeah, we understand the transition. But we want to combine our analysis with HIG-19-007 (HZaTo4L), which use the re-reco sample. Of course, we will also submit the Ultra Legacy request.
Since we need this sample for the pre-legacy sample and this process is 5 flavour scheme  and LO process. So, Could you recommend a set of pdf for 2016 and 2017-2018?